packages/styletron-react/package.json

```diff
@@ -6,27 +6,24 @@
   "repository": "styletron/styletron",
   "sideEffects": false,
   "files": [
-    "dist",
+    "dist-browser-cjs",
+    "dist-browser-esm",
+    "dist-node-cjs",
+    "dist-node-esm",
     "src",
-    "!__tests__"
+    "!__tests__",
+    "lib"
   ],
-  "main": "./dist/index.js",
-  "module": "./dist/index.es.js",
+  "main": "./dist-node-cjs/index.js",
+  "module": "./dist-node-esm/index.js",
+  "types": "./lib/index.d.ts",
   "browser": {
-    "./dist/index.js": "./dist/browser.es5.js",
-    "./dist/index.es.js": "./dist/browser.es5.es.js"
-  },
-  "es2015": {
-    "./dist/browser.es5.es.js": "./dist/browser.es2015.es.js"
-  },
-  "es2017": {
-    "./dist/browser.es5.es.js": "./dist/browser.es2017.es.js",
-    "./dist/browser.es2015.es.js": "./dist/browser.es2017.es.js"
+    "./dist-node-cjs/index.js": "./dist-browser-cjs/index.js",
+    "./dist-node-esm/index.js": "./dist-browser-esm/index.js"
```

Comment:
I'm missing some of the context on this and other packages entry point config. @rtsao should es2015/es2017 be maintained?

Reply:
this is a change because of create-universal-package v4